plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -729,16 +733,19 @@ func (p *antreaOctantPlugin) traceflowHandler(request service.Request) (componen
 		component.NewFormFieldHidden("action", runTraceAgainAction),
 	}}
 	runTraceAgain := component.Action{
-		Name: "Run Trace Again",
+		Name:  "Run Trace Again",
```

Comment:
It seems golangci-lint doesn't check this file somehow

Reply:
I'll add coverage. It makes sense because IIRC golangci-lint works on a single module and the Octant plugin is in its own module.

Reply:
Got it, thanks for the explanation.